test/Shared/UserManagerTestBase.cs

```diff
@@ -1093,39 +1096,45 @@ public Task<IdentityResult> ValidateAsync(UserManager<TUser> manager, TUser user
         public async Task CanQueryableRoles()
         {
             var manager = CreateRoleManager();
-            var roles = GenerateRoles("CanQuerableRolesTest", 4);
-            foreach (var r in roles)
+            if (!manager.SupportsQueryableRoles)
```

Comment:
I fixed this for queryable users missed this for roles. Was doing some refactoring using Resharper but thanks for catching